lib/internal/Magento/Framework/Url.php

```diff
@@ -93,6 +93,11 @@ class Url extends \Magento\Framework\DataObject implements \Magento\Framework\Ur
         '_scope_to_url',
     ];
 
+    /**
+     * @var \Magento\Store\Model\StoreManagerInterface
+     */
+    protected $_storeManager;
```

Comment:
Please, avoid usage underscore in the variables name and make the property private